src/App.tsx

```diff
@@ -35,6 +36,7 @@ function App() {
                 <Route path="/profile/edit-new-password" element={<EditNewPassword/>}/>
                 <Route path="/registration/user-id" element={<RegistrationUserID/>}/>
                 <Route path="/registration/name" element={<RegistrationName/>}/>
+                <Route path="/registration/email" element={<RegisterEmailPage/>}/>
```

Comment:
would it make sense to have these paths as constants? It would be less error prone if used elsewhere